src/app/components/UserMenu/index.tsx

```diff
@@ -42,54 +42,57 @@ export default function UserMenu() {
       console.error(e);
     }
   }
-
   return (
     <Menu as="div" className="relative">
       <Menu.Button className="flex items-center text-gray-700 dark:text-white hover:text-black dark:hover:text-white transition-colors duration-200">
         <MenuIcon className="h-6 w-6" />
       </Menu.Button>
       <Menu.List position="left">
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("discover");
-          }}
-        >
-          <GlobeIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("discover")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("publishers");
-          }}
-        >
-          <TransactionsIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("websites")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            navigate("/send");
-          }}
-        >
-          <SendIcon className="w-6 h-6 -ml-0.5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("actions.send")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            navigate("/receive");
-          }}
-        >
-          <ReceiveIcon className="w-6 h-6 -ml-0.5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("actions.receive")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("transactions");
-          }}
-        >
-          <TransactionsIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("transactions")}
-        </Menu.ItemButton>
-        <Menu.Divider />
+        {window.location.pathname !== "/options.html" && (
```

Comment:
Wouldn't it make more sense to rely on the screen width (instead of the page you are currently on)? You could also open the extension on your mobile phone.

Reply:
Option page have ton of different functionalities(setting, connected sites etc..) that needs more space.
Also, an extension can't run on mobile phone, so it won't be a problem here.

Reply:
I agree that there is still room for improvement in terms of responsive functionality in general, but extensions *can* run on mobile phones. 

Check out Kiwi Browser on Android (Safari on mobile support is also coming soon for Alby). I think it's important to have basic navigation work also on smaller screen sizes.

Reply:
Ohh that's good, I wasn't aware that extension works on mobile too.

Actually the idea of "removing fields from the hamburger menu" was to remove duplication as those options are already available on the `options` page. The extension automatically opens up `option` page in a new tab to show more functionalities that's inaccessible from the `popup`.

It was never about showing small menu on small screens but about removing duplicate menus.

![image](https://user-images.githubusercontent.com/66105983/230706968-1fe80d72-e887-45cc-87dc-b742425da882.png)

Reply:
If I currently open the extension on mobile I see this:  

![image](https://user-images.githubusercontent.com/100827540/230714747-46028712-0da7-443f-8b37-61371c358853.png)

With this change there is currently no way for me as a user to access main features like Send, Receive & Transactions on smaller screen resolutions.

Reply:
Ohh, so that's the issue. Will be pushing some more changes to fix it soon.

Reply:
Hey @reneaaron, just fixed this.